tracer/src/Datadog.Trace/ClrProfiler/ServerlessInstrumentation/AWS/LambdaCommon.cs

```diff
@@ -156,5 +154,19 @@ internal static void Flush()
                 Serverless.Error("Could not flush to the extension", ex);
             }
         }
+
+        internal static string SerializeObject<T>(T obj)
+        {
+            try
+            {
+                return JsonConvert.SerializeObject(obj);
+            }
+            catch (Exception e)
+            {
+                Serverless.Debug("Failed to serialize object with the following error:");
+                Serverless.Debug(e.ToString());
+                return JsonConvert.SerializeObject(DefaultJson);
```

Comment:
I think you want to return `DefaultJson` which already contains the string `"{}"`. If you serialize that string into JSON again, you end up with `"\"{}\""` (4 characters).

```suggestion
                return DefaultJson;
```

Reply:
Completely agree, nice catch!